app/main/routes.py

```
@@ -0,0 +1,95 @@
+        flash("Aufgabe nicht gefunden.", "error")
+    return redirect(url_for('main.home'))
+
+@main_bp.route('/generate-text', methods=['POST'])
+def generate_text():
+    prompt = request.form.get('prompt')
+    if prompt:
+        try:
+            text_generator = Text()
+            generated_text = text_generator(prompt)
+            session['generated_text'] = generated_text
+        except (ConnectionError, TimeoutError) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)